tests/integration_tests/test_pytorch_lightning.py

```diff
@@ -22,6 +22,7 @@
     from pytorch_lightning.strategies.ddp_spawn import DDPSpawnStrategy
     import torch
     from torch import nn
+    from torch.multiprocessing.spawn import ProcessRaisedException
```

Comment:
[Notes] I'm wondering why the test passed, but it was because `ProcessRaisedException` is imported at `torch/multiprocessing/__init__.py`.

https://github.com/pytorch/pytorch/blob/24ce3a7c3453734bc9870661b1e44d778c2a9384/torch/multiprocessing/__init__.py#L38-L39

We can see only four functions are exported at `torch.multiprocessing` (i.e., `'set_sharing_strategy', 'get_sharing_strategy', 'get_all_sharing_strategies'`).

https://github.com/pytorch/pytorch/blob/24ce3a7c3453734bc9870661b1e44d778c2a9384/torch/multiprocessing/__init__.py#L21-L22